spec/faraday/connection_spec.rb

```diff
@@ -453,8 +453,40 @@
       expect { conn.proxy = { uri: :bad_uri, user: 'rick' } }.to raise_error(ArgumentError)
     end
 
-    it 'gives priority to manually set proxy' do
+    it 'uses env http_proxy' do
+      with_env 'http_proxy' => 'http://proxy.com' do
```

Comment:
This one does not call itself env-proxy in the domain name. Is there a difference here?

Reply:
In the other tests, the `env-proxy.com` domain name helps distinguish the `http_proxy` setting from the `conn.proxy` setting that should be overshadowing it. 

This test is simply reading from the env, so there's no need to change it.